cluster/directory/static_directory.go

```diff
@@ -61,7 +61,7 @@ func (dir *staticDirectory) IsAvailable() bool {
 // List List invokers
 func (dir *staticDirectory) List(invocation protocol.Invocation) []protocol.Invoker {
 	l := len(dir.invokers)
-	invokers := make([]protocol.Invoker, l, l)
+	invokers := make([]protocol.Invoker, l)
```

Comment:
why delete capacity ?

Reply:
gosamle detect it. I also think the `capactity` no useful
```
static_directory.go:64:39: S1019: should use make([]protocol.Invoker, l) instead (gosimple)
invokers := make([]protocol.Invoker, l, l)
	                                     ^
```